website/README.md

```diff
@@ -415,26 +468,36 @@ Let's look at an example. Say you have a page called `/docs/foo` which needs to
 
 Finally, we run a global search for internal links to `/foo`, and make sure to adjust them to be `/nested/foo` - this is to ensure that client-side navigation still works correctly. _Adding a redirect alone is not enough_.
 
-One more example - let's say that content is being moved to an external website. A common example is guides moving to `learn.hashicorp.com`. In this case, we take all the same steps, except that we need to make a different type of change to the `docs-navigation` file. If previously the structure looked like:
-
-```js
-{
-  category: 'docs',
-  content: [
-    'foo'
-  ]
-}
+One more example - let's say that content is being moved to an external website. A common example is guides moving to `learn.hashicorp.com`. In this case, we take all the same steps, except that we need to make a different type of change to the `docs-nav-data` file. If previously the structure looked like:
+
+```json
+[
+  {
+    "name": "Docs",
+    "routes": [
+      {
+        "title": "Foo",
+        "path": "docs/foo"
+      }
+    ]
+  }
+]
 ```
 
-If we no longer want the link to be in the side nav, we can simply remove it. If we do still want the link in the side nav, but pointing to an external destnation, we need to slightly change the structure as such:
-
-```js
-{
-  category: 'docs',
-  content: [
-    { title: 'Foo Title', href: 'https://learn.hashicorp.com/<product>/foo' }
-  ]
-}
+If we no longer want the link to be in the side nav, we can simply remove it. If we do still want the link in the side nav, but pointing to an external destination, we need to slightly change the structure as such:
+
+```json
```

Comment:
Does this mean we should stop using the website/redirects.next.js file for these types of redirects?

Reply:
I think the redirects situation is still the same - if you were to change a link from internal to external (say, moving a getting started guide to Learn), then I think we'd still want to do both the sidebar change (as outlined here) and the redirect. To put it another way, I believe the `.js` => `.json` format change shouldn't affect anything related to redirects.